brainstormforce/astra
# AST-2854 - Visibility controls missing from header builder section & their dynamic CSS has display issue

### Description
- This PR address 2 issues
- 1. Visibility controls missing from header builder complete section
- 2. Visibility CSS has display CSS issue due to old option saved 
- The issue will clearly replicate with customizer JSON = https://share.getcloudapp.com/mXub2xnd
- PR - https://github.com/brainstormforce/astra/pull/4645
- Here header sections all visibility controls removed https://share.getcloudapp.com/12uRzLon only footer's are there which is why header's controls missed

### Screenshots


### Types of changes
- Bug fix

### How has this been tested?
- Switch to v3.9.4 theme
- Set any "Hide on Tablet" / "Hide on Mobile" option on & then make it off
- Now publish the customizer navigate to frontend
- Update the them / switch to master
- 1. You will see that header/module will disappear (as false value saved in DB by making we off previously)
- 2. Second issue is all visibility options from header section are missing

### Checklist:
